packages/cli/api-importers/v2-importer-commons/src/converters/ExampleConverter.ts

```diff
@@ -5,6 +5,7 @@ import { AbstractConverter, AbstractConverterContext, type OpenApiError } from "
 export declare namespace ExampleConverter {
     export interface Args extends AbstractConverter.Args<AbstractConverterContext<object>> {
         schema: OpenAPIV3_1.SchemaObject | OpenAPIV3_1.ReferenceObject;
+        generateOptionalProperties?: boolean;
```

Comment:
@eyw520 can we make sure that `generateOptionalProperties` is true for responses and false for requests?

Reply:
e.g. developers can see a minimal request but a full response